### DISCUSSION TITLE
我的AI阅读助手 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#141] [Announcements]
# 我的AI阅读助手 · BMPI

基于 chatGPT 的聊天机器人，帮助你高效阅读。

https://www.bmpi.dev/self/my-gpt-reader/

**Comment by dgqyushen** [comment]
大佬有兴趣做一个html版本？可以docker网页部署

**Reply by madawei2699** [reply]
为什么要造一个网页版？slack 多好用，比你搞一个独立app都好用

**Reply by dgqyushen** [reply]
我没用过slack，我试下去用hhh

**Comment by hh1985** [comment]
如果想用pubmed的文献摘要（人工做一些过滤）来训练一个机器人，帮着自己梳理研究思路，是不是也可以从GPT-index入手呢？

**Reply by hh1985** [reply]
当前有一个设想：用GPT将文献处理成一个知识图谱KG，然后再让GPT基于这个KG来进行问题的回答，这样结果的准确性应该会比较高吧？

**Reply by madawei2699** [reply]
gpt_index 好像有一个知识图谱的 reader，我还没研究过，感觉你的思路是可行的，可能要比单纯的相似度搜索更精准一些。

**Comment by ihzgnat** [comment]
点文中的slack超链接，显示此链接不再有效😓

**Reply by madawei2699** [reply]
更新了

**Reply by catwie** [reply]
有没有新的slack超链接，又失效了😭

**Reply by madawei2699** [reply]
更新了

**Reply by iamtj2006** [reply]
点文中的slack超链接，又失效了，无法加入了？麻烦再更新一下，如何添加到slack上体验一下了，谢谢。

**Reply by madawei2699** [reply]
已更新

**Comment by conge** [comment]
大家都在AI开辟的新区域学习游泳呀

**Comment by whisxixi** [comment]
请问如果提供的链接或者数据源包含较多内容，是否也可以一并处理？我在slack中试了一个链接，但是发现几个问题之后它似乎开始无法理解我的问题。

**Reply by madawei2699** [reply]
会话是有可能干扰的，这时候重开一个 thread 与它交流效果会更好。

**Comment by fryeggs** [comment]
dawei老师你好，我从你GITHUB上下载了，想在本地上部署一下，可是主文件里，没有启动的文件是咋回事？

**Comment by iamtj2006** [comment]
点文中的slack超链接，又失效了，无法加入了？麻烦再更新一下，如何添加到slack上体验一下了，谢谢。

**Comment by weiyi88** [comment]
dawei 老哥你好，有没有兴趣弄个分离版本，也就是说，chatGPT 功能为一个服务，slack 为一个服务。可以供开发者，去扩展，对接推特、tg、等等各类平台。这样会有更大的创造力哦

**Comment by xiaoshandegithub** [comment]
dawei老师，我想加入你的slack 频道来体验它的强大，麻烦更新一下链接
